docs/aws-iam.md

```diff
@@ -0,0 +1,54 @@
+# AWS IAM authentication
+
+!!! admonition "Version added: [4.4.19-19](release_notes/4.4.19-19.md)"
+
+!!! important
+
+    This is a [technical preview feature](glossary.md#technical-preview-feature)
+
+IAM (Identity Access Management) is the AWS service that allows you to securely control access to AWS resources. Percona Server for MongoDB supports authentication with AWS IAM enabling you to use the same AWS credentials both for it and other components of your infrastructure. This saves your DBAs from managing different sets of secrets and frees their time on other activities. 
+
+You can configure AWS IAM for a password-less authentication. Instead of username and password, the user or the application presents the secret key and the temporary credentials for authentication. The secret key is not sent to Percona Server for MongoDB thus significantly increasing the security in your infrastructure. For this scenario, configure the AWS instance that you use to support temporary credentials.
```

Comment:
```suggestion
You can configure AWS IAM for a password-less authentication. Instead of username and password, the user or the application presents AWS security credentials for authentication, but the secret key is not sent to Percona Server for MongoDB thus significantly increasing the security in your infrastructure.
```

Reply:
Done